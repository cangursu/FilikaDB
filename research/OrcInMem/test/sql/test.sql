
select orc_inmem_test('/home/postgres/sock_orcinmem');



