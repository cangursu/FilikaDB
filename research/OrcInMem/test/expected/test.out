select orc_inmem_test('/home/postgres/sock_orcinmem', '/home/postgres/projects/filikadb/research/OrcInMem/test/sql/data/');
     orc_inmem_test     
------------------------
 orc_inmem_test SUCCEED
(1 row)

